### DISCUSSION TITLE
博士论文送审要求添加图片中英文标注问题

### DISCUSSION THREAD

**Discussion by jiaqiwang969** [sjtug/SJTUThesis#854] [提问]
博士论文送审要求添加图片中英文标注，但如何使得英文标注不出现在图片标注目录

**Comment by LogCreative** [comment]
检查是否使用 `\bicaption` 命令添加中英文标注，此时英文标注不出现在插图索引。

https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/contents/floats.tex#L26-L30
